### PR TITLE
Generate CI on push to main branch

### PR DESCRIPTION
**Pull request by pierDipi**
Fixes https://github.com/openshift-knative/hack/issues/51

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,9 @@ include pkg/project/testdata/env
 generate-ci: clean generate-eventing-ci generate-serving-ci
 .PHONY: generate-ci
 
+generate-ci-no-clean: generate-eventing-ci generate-serving-ci
+.PHONY: generate-ci-no-clean
```
Comment (matzew) [inline Makefile:9]: no clean?

Comment (pierDipi) [reply Makefile:9]: The other one call `clean`, this one doesn't, hence "no clean"

Comment (pierDipi) [reply Makefile:9]: A better name? `without-clean` seems very long

Comment (matzew) [reply Makefile:9]: no clean is good, after this extra context.
thax

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**